packages/std/Cargo.toml

```diff
@@ -29,7 +29,7 @@ backtraces = ["snafu/backtraces"]
 
 [dependencies]
 base64 = "0.11.0"
-serde-json-wasm = "0.1.2"
+serde_json = { git = "https://github.com/webmaster128/serde_json_wasm", branch="float-feature" }
```

Comment:
Wow. I need to look at this.

Can you put this repo under confio or cosmwasm? Let me know if you need permissions.

Two main questions:
* it seems you recompiled the contract and it worked with check_wasm. Awesome!
* gas was not affected significantly:+1:
* this increased contract size by 70kb. I wonder if you can do some size profiling. I used twilly on the output before wasm-opt. Eg just compile with the rustflags argument and see what is taking up the space... maybe there is something easy to trim out

Reply:
The current state is that `cargo wasm` works perfectly but `cargo +nightly wasm` still has disallowed instructions in it. I think it is related to how the different compilers handle the many `#[inline]` statements from the 'serde-rs/json'. I'm in the process of kicking out much more code. Let's see